lib/src/metrics/nesting_level/nesting_level_visitor.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:analyzer/source/line_info.dart';
+
+class NestingLevelVisitor extends RecursiveAstVisitor<Object> {
```

Comment:
RecursiveAstVisitor isn't returning anything, so Object type could be replaced with void

Reply:
fixed